src/app/collective-rewards/allocations/context/AllocationsContext.tsx

```diff
@@ -153,12 +153,13 @@ export const AllocationsContextProvider: FC<{ children: ReactNode }> = ({ childr
       acc[builder.address] = {
         ...builder,
         backerRewardPercentage: {
-          active: backerRewards[index]?.active ?? BigInt(0),
-          previous: backerRewards[index]?.previous ?? BigInt(0),
-          next: backerRewards[index]?.next ?? BigInt(0),
-          cooldown: backerRewards[index]?.cooldown ?? BigInt(0),
+          active: BigInt(backerRewards[index]?.active ?? 0n),
```

Comment:
```suggestion
          active: backerRewards[index]?.active ?? 0n,
```
I don't think that we should do the `BigInt(backerRewards[index]?.active ?? 0n)`, with `backerRewards[index]?.active ?? 0n` is enough, viem already parse it to bigint or even us before. 

This applies for all of the the places that are wrapped by BigInt()